corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -59,7 +59,7 @@ def get_forms(form_ids, ordered=False):
 
     @staticmethod
     def get_form_ids_in_domain_by_type(domain, type_):
-        pass
+        return get_form_ids_by_type(domain, type_)
```

Comment:
currently this is only used for SQL domains hence why it hasn't been an issue but thought I'd implement it anyway